src/sentry/api/bases/organization.py

```diff
@@ -146,6 +146,10 @@ class OrganizationSearchPermission(OrganizationPermission):
     }
 
 
+class OrganizationDataExportPermission(OrganizationPermission):
+    scope_map = {"GET": ["event:read", "event:admin"], "POST": ["event:read", "event:admin"]}
```

Comment:
This essentially just gives access to event data in a different form, so I figured having the POST and GET endpoints scoped both to `event:read` makes sense. The POST requests just create files from data the user can already see.

Reply:
@leeandher So your endpoints will have different permission requirements than the legacy data export endpoints?

Reply:
Nvm, I see you changed the legacy ones as well